contracts/credit/CreditFacadeV3.sol

```diff
@@ -394,6 +388,8 @@ contract CreditFacadeV3 is ICreditFacadeV3, ACLNonReentrantTrait {
         whenNotExpired // U:[FA-3]
         nonReentrant // U:[FA-4]
     {
+        _getBorrowerOrRevert(creditAccount); // U:[FA-5]
```

Comment:
Absence of this could have been a problem with special bots (for normal bots, `permissions != 0` implied account existence).